app/plugins/ee-openid-connect.md

```diff
@@ -63,24 +63,21 @@ The plugin has been tested with several OpenID Connect capable providers such as
 - [Keycloak][keycloak]
 - [Microsoft Azure Active Directory v1][azurev1]
 - [Microsoft Azure Active Directory v2][azurev2]
-- Microsoft Live Connect
+- [Microsoft Live Connect][live]
 - [Okta][okta]
 - [OneLogin][onelogin]
 - [OpenAM][openam]
 - [Paypal][paypal]
 - [PingFederate][pingfederate]
-- [Salesfoce][salesforce]
+- [Salesforce][salesforce]
 - [Yahoo!][yahoo]
 
-As long as your provider supports OpenID Connect standards the plugin should
-work, even if it is not specifically tested against it. Let us know if you
-want your provider to be tested and added to the list.
+As long as your provider supports OpenID Connect standards (exposing a {service_url}.well-known/openid-configuration endpoint) the plugin should work, even if it is not specifically tested against it. Let us know if you want your provider to be tested and added to the list.
```

Comment:
Should the link here be enclosed in code blocks for readability?

(exposing a `{service_url}.well-known/openid-configuration` endpoint)

Reply:
We should also respect the 80 chars limit @bungle tried to enforce here... Tons of plugins will do this automatically for various text editors.

Reply:
fixing and will re-submit